### PR TITLE
Convert package lists into arrays

### PR DESCRIPTION
**Pull request by scollazo**
This avoids a warning, and is also faster than using a loop

### CODE REVIEW

File: tasks/install-centos.yml [hunk 1]
```diff
@@ -7,14 +7,12 @@
 
 - name: "Install percona database server"
   yum:
-    name: "{{ item }}"
+    name: [ 'Percona-Server-server-{{mysql_version_major}}{{mysql_version_minor}}',
+            'Percona-Server-client-{{mysql_version_major}}{{mysql_version_minor}}',
+            'Percona-Server-devel-{{mysql_version_major}}{{mysql_version_minor}}',
+            'percona-toolkit',
+            'percona-xtrabackup' ]
```
Comment (archivero-madrid) [inline tasks/install-centos.yml]: i prefer the old style
why to change?

- name: "Install percona database server"
  yum:
    name:
      - 'Percona-Server-server-{{mysql_version_major}}{{mysql_version_minor}}'
      - 'Percona-Server-client-{{mysql_version_major}}{{mysql_version_minor}}'
      - 'Percona-Server-devel-{{mysql_version_major}}{{mysql_version_minor}}'
      - 'percona-toolkit'
      - 'percona-xtrabackup'

Comment (scollazo) [reply tasks/install-centos.yml]: Due to ansible changes, package installs using loops is deprecated:  https://docs.ansible.com/ansible/latest/porting_guides/porting_guide_2.7.html#using-a-loop-on-a-package-module-via-squash-actions

Comment (archivero-madrid) [reply tasks/install-centos.yml]: the data type is still a list, that has not changed

you changed the sequence style but i don't understand why

in yaml
```
a: [1,2,3]
```

and
```
a:
- 1
- 2
- 3
```

is the same

**Comment by archivero-madrid** [comment]
"Convert package lists into arrays"
 ---> lists and arrays are the same

**Pull request merged**